ddtrace/tracer/tracer.go

```diff
@@ -322,6 +323,7 @@ func (t *tracer) StartSpan(operationName string, options ...ddtrace.StartSpanOpt
 		// this is a brand new trace, sample it
 		t.sample(span)
 	}
+	log.Debug("Started Span: %v, Operation: %s, Resource: %s, Tags: %v, %v", span, span.Name, span.Resource, span.Meta, span.Metrics)
```

Comment:
This format is complicated and it isn't clear what exactly the output will be like.
We should add a test to `ddtrace/tracer/log_test.go` that tests this output to ensure the format continues to make sense in the future.